### DISCUSSION TITLE
Support Google Consent mode in gatsby-plugin-google-tagmanager

### DISCUSSION THREAD

**Discussion by aileenrose** [gatsbyjs/gatsby#28988] [Ideas / Feature Requests]
Allow script tag to be updated with `data-cookieconsent="ignore"` to support the Google Consent mode and cookie consent management apps like Cookiebot.

https://support.cookiebot.com/hc/en-us/articles/360009192739-Google-Tag-Manager-and-Automatic-cookie-blocking

**Comment by vanska** [comment]
@aileenrose If still relevant, I made a plugin for Cookiebot that does just this https://github.com/nitrofi/gatsby-plugin-cookiebot
